components/skin/ui/src/main/resources/PhenoTips/ContentTopMenu.xml

```diff
@@ -891,7 +891,9 @@ $xwiki.ssx.use('PhenoTips.JSONImportDialog')##
 ** (% class="empty" %)----
 ** [[$services.localization.render('phenotips.contentTopMenu.moreActionsMenu.exportPDF')&gt;&gt;path:$doc.getURL('export', "format=pdf&amp;pdfcover=0&amp;pdftoc=0&amp;pdftemplate=${pdfTemplate}")|| class="export-link"]]
 #if($isAdvancedUser)
+#if ($hasAdmin)
```

Comment:
I wonder if the user has to be both an admin and an advanced user to get this entry in the menu. In other words, should the `if` above be moved after this `if`?